cityofaustin/atd-moped
# Show component work types in component list items

## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/19560

## Testing
**URL to test:** 
https://deploy-preview-1479--atd-moped-main.netlify.app/moped/projects/2049?tab=map&project_component_id=2686

**Steps to test:**

Expand components and see the work type in a gray chip

![Screenshot 2024-11-11 at 8 45 30 AM](https://github.com/user-attachments/assets/139c687d-124b-43f2-83bf-fce2b4a08c26)


---
#### Ship list
   - Added to component focus test - "You should see chips with the component works types in the expanded list item."
